vars/writeBuildInfo.groovy

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/groovy
+
+import groovy.json.JsonBuilder
+import groovy.json.JsonSlurperClassic
+import org.feedhenry.Utils
+
+def call(name, versionTxt) {
+    def utils = new Utils()
+    def buildInfoFileName = utils.getBuildInfoFileName()
+    def buildInfo = [:]
+
+    if(fileExists(buildInfoFileName)) {
+        def buildInfoRaw = readFile buildInfoFileName
+        print buildInfoRaw
+        buildInfo = new JsonSlurperClassic().parseText buildInfoRaw
+        print buildInfo
+    }
+
+    buildInfo['jenkinsUrl'] = env.JENKINS_URL
+    buildInfo['buildUrl'] = env.BUILD_URL
+    buildInfo['sha1'] = sh(returnStdout: true, script: "git log -n 1 --pretty=format:\"%H\"").trim()
```

Comment:
Yes i guess the outer quotes could be single, probably had a variable in there once upon a time from wherever i copied it.